src/functionalTest/resources/features/F-007/S-063.td.json

```diff
@@ -88,7 +88,6 @@
             "substantive": "Y",
             "primaryLocation": "[[ANYTHING_PRESENT]]",
             "jurisdiction": "CIVIL",
-            "region": "2",
```

Comment:
As this change is dependent on the new flag we should look to update the `@FeatureToggle` for S-063 in `/src/functionalTest/resources/features/F-007/F-007.feature`, i.e. AAT and Master pipelines would fail this test if new flag not turned on in AAT/Staging.

Reply:
i have updated the S-063 @FeatureToggle in F-007.feature to

@FeatureToggle(DB:civil_wa_1_4=on)

please check that this looks correct

Reply:
Resolved in 0b65e6dae1ba2ed4f688252274b7dfb119ad6e0f.